static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -17,9 +21,19 @@
     }
 }
 
+#headlines.fc-container {
```

Comment:
Is there a way of targetting this element without using the `id`? The specificity of this selector looks a bit heavy